terraform/psn/psn.tf

```diff
@@ -17,61 +17,3 @@ variable "security_group_name" {
   type        = string
   description = "The security group to allow access to the PSN VPC Endpoint."
 }
-
-data "aws_security_group" "security_group" {
-  name = var.security_group_name
-}
-
-resource "aws_vpc_endpoint" "psn_service" {
-  vpc_id            = var.vpc_id
-  service_name      = var.vpc_endpoint
-  vpc_endpoint_type = "Interface"
-
-  security_group_ids = [aws_security_group.psn_endpoint.id]
-
-  subnet_ids          = var.subnet_ids
-  private_dns_enabled = false
-}
-
-resource "aws_security_group" "psn_endpoint" {
-  name        = "psn-endpoint"
-  description = "The PSN VPC Endpoint"
-  vpc_id      = var.vpc_id
-}
-
-resource "aws_security_group_rule" "psn_ingress_from_cells" {
-  security_group_id = aws_security_group.psn_endpoint.id
-
-  type      = "ingress"
-  protocol  = "tcp"
-  from_port = 3128
-  to_port   = 3128
-
-  source_security_group_id = data.aws_security_group.security_group.id
-}
-
-resource "aws_security_group_rule" "cells_egress_to_psn" {
-  security_group_id = data.aws_security_group.security_group.id
-
-  type      = "egress"
-  protocol  = "tcp"
-  from_port = 3128
-  to_port   = 3128
-
-  source_security_group_id = aws_security_group.psn_endpoint.id
-}
-
-data "aws_network_interface" "psn_interface" {
-  for_each = aws_vpc_endpoint.psn_service.network_interface_ids
-
-  id = each.value
-}
-
-output "psn_security_group_seed_json" {
```

Comment:
We should validate that the outputs aren't required further down the pipeline

Reply:
Good spot. I've updated I will check how it was being used.